src/MSBuild.Conversion.Project/ProjectRootElementExtensionsForConversion.cs

```diff
@@ -39,8 +40,12 @@ public static IProjectRootElement RemoveDefaultedProperties(this IProjectRootEle
             foreach (var propGroup in projectRootElement.PropertyGroups)
             {
                 var configurationName = MSBuildHelpers.GetConfigurationName(propGroup.Condition);
-                var propDiff = differs[configurationName].GetPropertiesDiff();
-
+                if (!differs.ContainsKey(configurationName))
+                {
+                    continue;// Este: temporary fix, what is differs and why does it still have a reference to the old removed properties for winui?
```

Comment:
This is likely because it read a model of the old project file and still has those old properties. Unless you feed it a modified old-style project with those removed, the old properties will still be present.

A `differ` is an object that maintains the diff of properties and items between two MSBuild project evaluations:

* Project A has XYZ properties and ABC items
* Project B has WXY properties and ABD items

The differ will tell us where each has a difference.

Reply:
👍